lib/dotcom_web/components/trip_planner/itinerary_group.ex

```diff
@@ -2,47 +2,146 @@ defmodule DotcomWeb.Components.TripPlanner.ItineraryGroup do
+      >
+        <%= @summary.tag %>
+      </div>
+      <div class="flex flex-row mb-3 font-bold text-lg justify-between">
+        <div>
+          <%= format_datetime_full(@summary.first_start) %> - <%= format_datetime_full(
+            @summary.first_stop
+          ) %>
+        </div>
+        <div>
+          <%= @summary.duration %> min
+        </div>
+      </div>
+      <div class="flex flex-wrap gap-1 items-center content-center mb-3">
+        <%= for {summary_leg, index} <- Enum.with_index(@summary.summarized_legs) do %>
+          <.icon :if={index > 0} name="angle-right" class="font-black w-2" />
+          <.leg_icon {summary_leg} />
+        <% end %>
+      </div>
+      <div class="flex flex-wrap gap-1 items-center mb-3 text-sm text-grey-dark">
+        <div :if={@summary.accessible?} class="inline-flex items-center gap-0.5">
+          <.icon type="icon-svg" name="icon-accessible-small" class="h-3 w-3 mr-0.5" /> Accessible
+          <.icon name="circle" class="h-0.5 w-0.5 mx-1" />
+        </div>
+        <div class="inline-flex items-center gap-0.5">
+          <.icon name="person-walking" class="h-3 w-3" />
+          <%= @summary.walk_distance %> mi
+        </div>
+        <div :if={@summary.total_cost > 0} class="inline-flex items-center gap-0.5">
+          <.icon name="circle" class="h-0.5 w-0.5 mx-1" />
+          <.icon name="wallet" class="h-3 w-3" />
+          <%= Fares.Format.price(@summary.total_cost) %>
+        </div>
+      </div>
+      <div class="flex justify-end items-center">
+        <div :if={Enum.count(@summary.next_starts) > 0} class="grow text-sm text-grey-dark">
+          Similar trips depart at <%= Enum.map(@summary.next_starts, &format_datetime_short/1)
+          |> Enum.join(", ") %>
+        </div>
+        <button class="btn-link font-semibold underline" phx-click={JS.toggle(to: "##{@group_id}")}>
+          Details
+        </button>
+      </div>
+      <div id={@group_id} class="mt-30" style="display: none;">
+        <.itinerary_detail :for={itinerary <- @itineraries} itinerary={itinerary} />
+      </div>
     </div>
     """
   end
 
+  attr(:class, :string, default: "")
+  attr(:routes, :list, required: true, doc: "List of %Routes.Route{}")
+  attr(:walk_minutes, :integer, required: true)
+
+  # No routes
+  defp leg_icon(%{routes: [], walk_minutes: _} = assigns) do
+    ~H"""
+    <span class={[
+      "flex items-center gap-1 text-sm font-semibold leading-none whitespace-nowrap py-1 px-2 rounded-full border border-solid border-gray-light",
+      @class
+    ]}>
+      <.icon name="person-walking" class="h-4 w-4" />
+      <span><%= @walk_minutes %> min</span>
+    </span>
+    """
+  end
+
+  # Group of commuter rail routes are summarized to one symbol.
+  defp leg_icon(%{routes: [%Routes.Route{type: 2} | _]} = assigns) do
```

Comment:
Non-blocking, partly because I think my ideal outcome here might be a fairly large restructuring of how routes and modes are represented in dotcom (which is _definitely_ too much scope for this PR), but I feel like there are a lot of places where I have to memorize what `type: 2`/`type: 4`/etc mean in order to understand our code.